guides/main_concepts/relevancy.md

```diff
@@ -161,38 +157,36 @@ To add this newly created rule to the existing ranking rule, using the [settings
   "wordsPosition",
   "exactness",
-  "desc(release_date)",
-  "asc(movie_ranking)"
+  "asc(release_date)",
+  "dsc(movie_ranking)"
 ]
 ```
 
-## Attributes importance
+## Importance of the attributes
 
 In a dataset, some fields are more relevant to the search than others. A `title`, for example, has a value more meaningful to a movie search than its `description` or its `director` name.
 
-By default, MeiliSearch will determine the order of importance of the attributes based on the order in which they appear in the first document added. Then, each new attribute present in new documents will be added at the end of this ordered list.
-
-If you wish to specify the order of the attributes you can either define them in the settings, or set the correct order on the first document indexed in MeiliSearch.
+By default, the order of importance of the attributes is based on their order of appearance in the first document added. Then, each new attribute found in new documents will be added at the end of this ordered list.
 
-### Changing the attributes order
+If you wish to specify the order of the attributes you can either define them in the settings or set the correct order in the first document indexed.
 
-Possibly, you want to change the order after the documents have been added. This is still very possible.
+### Changing the order of the attributes
 
-When a document is added to MeiliSearch, every new attribute inside will be added to two lists :
+You may want to change the order once the documents have been ingested. This is still very possible using the searchable attributes list.
 
-- The [searchable attributes list](/references/searchable_attributes.md): attributes in which to search for matching query words.
-- The [displayed attributes list](/references/displayed_attributes.md): attributes in a document that are shown to the user.
+Whenever a document is added to MeiliSearch, all new attributes found in it are automatically added to two lists:
 
-The one that concerns this section is the searchable attributes list.
+- **The [searchable attributes list](/references/searchable_attributes.md)**: Attributes of the fields in which to search for matching query words.
+- **The [displayed attributes list](/references/displayed_attributes.md)**: Attributes of the fields displayed in documents.
 
-This list is **ordered**. This means that the order in which the attributes appear in the list will determine their relevancy. The earlier they appear in that list, the more important they are.
+This searchable attributes list is **ordered**, which means the order in which the attributes appear in the list determines their relevancy. Attributes are arranged from the most important attribute to the least important attribute.
 
-To change this order, you need to send the sorted-list, in the order you want, using the settings route.
+Place the attributes in the desired order and send this updated list using the settings routes. Attributes will be re-ordered.
```

Comment:
Could you add the link to setting route on "settings routes". 
```suggestion
Place the attributes in the desired order and send this updated list using the settings routes. Attributes will be re-ordered.
```

Reply:
Sure!